net/monero-wire/Cargo.toml

```diff
@@ -6,6 +6,9 @@ license = "MIT"
 authors = ["Boog900"]
 repository = "https://github.com/SyntheticBird45/cuprate/tree/main/net/monero-wire"
 
+[features]
+default = []
+tracing = ["levin-cuprate/tracing"]
```

Comment:
Not related to this PR but we should set all of our crates to `cuprate-*` eventually.

Reply:
The naming convention I have been using is: `*-cuprate` when a crate is not specific to Cuprate and `cuprate-*` when a crate is specific to Cuprate. 

I could rename this to `levin-codec` later, which would probably be clearer. I think this is the only crate which is `*-cuprate`.